internal/controller/streams.go

```diff
@@ -211,18 +210,28 @@ func (sm *streamManager) StreamLogs(ctx context.Context) error {
 	return nil
 }
 
-// StreamCiliumNetworkFlows handles the cilium network flow stream.
-func (sm *streamManager) StreamCiliumNetworkFlows(ctx context.Context, ciliumNamespace string) error {
+// findHubbleRelay returns a *CiliumFlowCollector if hubble relay is found in the given namespace
+func (sm *streamManager) findHubbleRelay(ctx context.Context, ciliumNamespace string) (*CiliumFlowCollector, bool) {
```

Comment:
Remove the `bool`.
If Hubble Relay is not found, this already returns a `nil` pointer. So the `bool` is redundant.